src/main/resources/words_be.properties

```diff
@@ -0,0 +1,44 @@
+GREETING_TEXT = Бясплатна разлічу кошт утыльзбору ў Беларусі для легкавога транспарту (катэгорыя М1).\n \
```

Comment:
usually keys in localization files have the format like this`answers.users.greeting`

Reply:
Keys were renamed